ConwaysGameOfLife.Tests/GeneratorShould.cs

```diff
@@ -1,10 +1,75 @@
 using System;
+using System.Collections.Generic;
 using Xunit;
 
 namespace ConwaysGameOfLife.Tests
 {
     public class GeneratorShould
     {
-        
+        [Theory]
+        [InlineData("XXX", "XXX", "XXX")]
+        [InlineData("XX", "XX", "XX")]
+        [InlineData("XXXX", "XXXX", "XXXX")]
+        public void ReturnNewListOfCells_FromOldUniverse(params string[] rows)

```

Comment:
Is this test required? When testing the `GenerateNewUniverse` function what matters is that when given an old universe the correct new universe is generated. When checking that the universes are equal in the later tests the universe helper function `UniversesAreEqual` is checking the list of cells anyway.